include/wlr/types/wlr_cursor.h

```diff
@@ -153,6 +153,15 @@ void wlr_cursor_map_to_region(struct wlr_cursor *cur, struct wlr_box *box);
 void wlr_cursor_map_input_to_region(struct wlr_cursor *cur,
 	struct wlr_input_device *dev, struct wlr_box *box);
 
+/**
+ * Locks the cursor to its current position
+ */
```

Comment:
I'm not sure that justification makes any sense. wlr_cursor does not move itself in response to input events. The compositor has to explicitly ask it to do so, and can just stop doing so when it wants to be locked. But it can still process input events in that situation...

Reply:
This is still here?